pkgs/applications/video/cutecapture/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, libusb1
+, sfml
+, writeShellScript
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cutecapture";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "Gotos";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "cutecapture";
```
If pname is appended this is wrong